generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -319,7 +320,10 @@ public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
                 // each scope as a GrantedAuthority, which we don't care about.
                 if (authority instanceof OidcUserAuthority) {
                     OidcUserAuthority oidcUserAuthority = (OidcUserAuthority) authority;
-                    mappedAuthorities.addAll(SecurityUtils.extractAuthorityFromClaims(oidcUserAuthority.getUserInfo().getClaims()));
+                    OidcUserInfo userInfo = oidcUserAuthority.getUserInfo();
+                    if (userInfo != null) {
```

Comment:
In which scenario you should get null over here? Isn't the UserInfo endpoint a requirement of `OIDC`?

Reply:
Spring Security loads the user info endpoint only if `email` or `profile` scope is provided (and if some other conditions are met).
Like I wrote in the description, I've connected our App to our ADFS. Unfortunately the ADFS does only return `odic` as scope and therefore Spring Security skips the user info resulting in a NPE.

This does not break anything, just wrapping it in a check.

Reply:
We send [`oidc,profile,email`](https://github.com/jhipster/generator-jhipster/blob/main/generators/server/templates/src/main/resources/config/application.yml.ejs#L296) as scopes. Are you saying ADFS doesn't work with this? It sounds like they're not OIDC-compliant if this is the case.

Reply:
Unfortunately yes. The ADFS has several quirks regarding OIDC (which I also stumbled upon googling for a solution).

Reply:
IMO, this is a useless check.